### DISCUSSION TITLE
pling.com - percepat timer download

### DISCUSSION THREAD

**Discussion by FakeAdsCollection** [realodix/AdBlockID#1653] [Personal Filter]
### Sample URL

https://www.pling.com/p/1403328/

### Ad Blocker

uBlock Origin

### Keterangan

bisa gak buatkan filterlist untuk mempercepat timer di website pling.com? (ini website international jangan ditambahkan ke adblockid. hanya untuk my filters.)
![image](https://github.com/realodix/AdBlockID/assets/73978235/691f0c2e-3b6a-45c3-8d1f-79713074c617)

**Comment by realodix** [comment]
Halo @FakeAdsCollection 

Pakai ini 

```adblock
pling.com##+js(adjust-setTimeout, Download, *, 0.001)
```
